src/pages/HistoryPage/EpochForm.tsx

```diff
@@ -422,121 +682,242 @@ const EpochForm = ({
-                    )}
-                  />
-                </Flex>
-                <Flex css={{ maxWidth: '150px' }}>
-                  <FormInputField
-                    id="days"
-                    name="days"
-                    defaultValue={
-                      source?.epoch?.days ?? source?.epoch?.calculatedDays ?? 4
-                    }
-                    control={control}
-                    label="Duration (days)"
-                    infoTooltip="How long the epoch lasts in days"
-                    number
-                  />
+                      >
+                        <FormTimePicker
+                          id="start_time"
+                          name="start_time"
+                          control={control}
+                          disabled={shouldFormBeDisabled}
+                        />
+                      </Box>
+                      <Text size="medium">
+                        In your
+                        <br /> local timezone
+                      </Text>
+                    </Flex>
+                  </Flex>
                 </Flex>
-                <Flex column css={{ gap: '$xs' }}>
-                  <FormLabel type="label" css={{ fontWeight: '$bold' }}>
-                    Start Time{' '}
-                    <Tooltip content="The start time of the epoch in your local time zone">
-                      <Info size="sm" />
-                    </Tooltip>
-                  </FormLabel>
-                  <Flex row css={{ gap: '$sm' }}>
+                <Flex
+                  column
+                  css={{
+                    display:
+                      getValues('repeat_view') === 'repeats' ? 'flex' : 'none',
+                    flexWrap: 'wrap',
+                    gap: '$md',
+                  }}
+                >
+                  <Flex
+                    css={{
+                      gap: '$xs',
+                    }}
+                  >
                     <Controller
                       control={control}
-                      name="start_date"
-                      render={({ field: { onChange, value, onBlur } }) => (
-                        <Box
-                          css={{
-                            maxWidth: '150px',
-                            '> div': { mb: '0 !important' },
-                          }}
-                        >
-                          <FormTimePicker
-                            onBlur={onBlur}
-                            onChange={onChange}
-                            value={value}
-                            disabled={
-                              selectedEpoch &&
-                              currentEpoch?.id === selectedEpoch?.id
-                            }
-                          />
-                        </Box>
+                      name="repeat"
+                      defaultValue="custom"
+                      render={({ field: { onChange, value } }) => (
+                        <Select
```

Comment:
Near impossible to see the dropdown in dark mode , not sure how we tweak this exactly @wingfeatherwave  
<img width="376" alt="image" src="https://user-images.githubusercontent.com/95149165/214372744-d74b20a8-ff01-4a0d-b95d-562c6bf6eed6.png">

Reply:
this has been resolved in main